### ISSUE TITLE
`Condition` event should not come back to the timeline it got called from

### ISSUE THREAD

**Issue by Crystalwarrior** [AnidemDex/Godot-EventSystem#34]
### Describe the project you are working on

Point-and-click investigation segment in a game where you have to find all objects to be able to leave the room.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

We cannot look at the timeline system as an extension of a scripting language - instead, the most comparable reference we have is the Assembly language. In ASM, there's a 'Conditional Jump' and 'Unconditional Jump'. Unconditional Jump equivalent in the event system is the "Change Timeline" and "Jump To", while Conditional event behaves like a standard GDScript IF-ELSE.

There is a big problem with this behavior, however. Without a background in programming, this behavior is _not obvious._ It is more intuitive to understand that when a condition is successful, it will JUMP to that targeted timeline, and since you do not explicitly say to "come back to previous timeline", then it shouldn't come back.

Otherwise, having large if-else chains with the timelines jumping back and forth is really easy to get lost in and is completely unintuitive, especially since in the current interface, unlike scripting, you do not see all the code at once, and instead have to jump back and forth.

### Adittional information

Simply put, "condition true: go to target timeline, condition false: ignore this event".
False timeline could still be optional to include, meaning the only real change this is suggesting is to simply "change timeline" to the condition.

**Comment by Crystalwarrior** [comment]
https://github.com/AnidemDex/Godot-EventSystem/pull/33
https://github.com/AnidemDex/Godot-EventSystem/issues/31

**Comment by Crystalwarrior** [comment]
Assembly language conditionals: https://www.tutorialspoint.com/assembly_programming/assembly_conditions.htm

**Comment by AnidemDex** [comment]
`Condition` events doesn't return to the original timeline in 1.4.3
